### ISSUE TITLE
Update Hunting Service Pricing Model

### ISSUE THREAD

**Issue by jimbrig** [lakecountryhuntclub/operations#4]
- **Assigned To:** @jimbrig 

- **Todo:** Complete steps outlined below and add new pricing for "Hunting Service Bundle Product" with a single option: *Number of Hunts* and prices vary accordingly.

- **Future Todos:** Discuss what other hunting or non-hunting services to offer as "products". The more content we can get sooner the better an initial site launch will be (from an SEO and Marketing standpoint especially). For example, some ideas are:

  - Add hunts for other game such as dove, deer, etc.
  - Add fishing expedition services
  - Add butterfly catching experiences (since this is tracked by NDR also)
  - Add Apparel, Gear, etc. With Branding for a physical store.
  - Add options to just rent dogs, etc.
  - See if NDR can accommodate us serving as an official retailer for Hunting Licenses and/or Stamps.

***

## Background

Currently, setup to accommodate hunts by longevity (hours), and extra services for Guide and Dog:

![image](https://github.com/lakecountryhuntclub/operations/assets/32652297/14cf1e4f-5133-4f37-a394-e178934fad8d)

with a pricing model based on these parameters (i.e. see below for example 2-hour hunt options):

| Price ($) | Duration (Hrs) | With Dog? | With Guide? |
| :----:    | :-----:        | :-----:   | :-----:     |
| $30       | 2 Hours        | NO        | NO          |
| $50       | 2 Hours        | YES       | NO          |
| $50       | 2 Hours        | NO        | YES         |
| $70       | 2 Hours        | YES       | YES         |

In other words, the base price for the hunt without any extra services is $30. Then if you want to add just dog or just guide it's an extra $20, or both for an extra $40. Its not logical but its how I set things up in Shopify to keep the product-as-a-service model in tact (not very easy on Shopify which is more suited for physical products)

## Next Actions

Should add a "Hunting Bundle" product where it is assumed that the hunts will have both a dog and guide and last for 2 hours and the pricing of the bundle follows the model from the business card:

From the design-kit repo's Business Cards PDF:

![image](https://github.com/lakecountryhuntclub/operations/assets/32652297/f1054d22-930b-4ce6-86e8-273fc1da700c)
